core/templates/pages/exploration-editor-page/exploration-save-and-publish-buttons/exploration-save-and-publish-buttons.component.spec.ts

```diff
@@ -269,4 +263,34 @@ describe('Exploration save and publish buttons component', function() {
     spyOn(editabilityService, 'isTranslatable').and.returnValue(false);
     expect($scope.isEditableOutsideTutorialMode()).toBe(false);
   });
+
+  it('should display publish button', function() {
```

Comment:
`should enable publish button when ....`

Reply:
DONE